Dockerfile

```diff
@@ -23,7 +23,7 @@ RUN apt-get update -y \
     && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
 # Install Docker
-RUN apt-get update && \
+RUN apt-get update -y && \
```

Comment:
I don't think you need to install `containerd.io` if you're just running sibling containers. This seems to imply that you're intending to run the Docker daemon inside a Docker container (aka Docker-in-Docker).

Reply:
It's needed for running on a Mac

Reply:
Hmm... So on a Mac you're running Docker inside Docker?

Reply:
I'm not sure what's going on with that. What I do know is that the command line to run the images needs to remain the same (with `-v /var/run/docker.sock:/var/run/docker.sock`) but that install is needed.

Reply:
Hmm... OK.